gufe/ligandnetwork.py

```diff
@@ -65,9 +66,16 @@ def graph(self) -> nx.MultiDiGraph:
             graph = nx.MultiDiGraph()
             # set iterator order depends on PYTHONHASHSEED, sorting ensures
             # reproducibility
-            for node in sorted(self._nodes):
```

Comment:
Can you maybe add to this node saying that we sort by inchikey and it's still succeptible to clashes on molecules that are the same but with different properties?

Also please do raise an issue for a follow-up on this.